requirements.txt

```diff
@@ -20,3 +20,4 @@ sentry-kafka-schemas==2.1.2
 sentry-sdk>=2.36.0
 sortedcontainers>=2.4.0
 typing-extensions>=4.15.0
+zipfile-zstd==0.0.4
```

Comment:
It looks like we get zstd for free in Python 3.14, not sure if you looked into that option: https://docs.python.org/3/library/compression.zstd.html#

Reply:
i tried upgrading us to 3.14, but that requires upgrading `confluent-kafka` to 2.12 which then conflicts with `sentry-arroyo` which has that package pinned to <2.10.0... so not possible right now

hen we're off of arroyo though.. lol